src/components/TopBar/TopBar.vue

```diff
@@ -142,27 +142,15 @@
 			<ExtendOneToOneDialog v-else-if="!isSidebar && canExtendOneToOneConversation"
 				:token="token" />
 
-			<!-- Reactions menu -->
-			<ReactionMenu v-if="isInCall && hasReactionSupport"
-				:token="token"
-				:supported-reactions="supportedReactions"
-				:local-call-participant-model="localCallParticipantModel" />
-
-			<!-- Local media controls -->
-			<TopBarMediaControls v-if="isInCall"
-				:token="token"
-				:model="localMediaModel"
-				:is-sidebar="isSidebar"
-				:local-call-participant-model="localCallParticipantModel" />
-
 			<!-- TopBar menu -->
 			<TopBarMenu :token="token"
 				:show-actions="!isSidebar"
 				:is-sidebar="isSidebar"
-				:model="localMediaModel"
 				@open-breakout-rooms-editor="showBreakoutRoomsEditor = true" />
 
-			<CallButton shrink-on-mobile :hide-text="isSidebar" :is-screensharing="!!localMediaModel.attributes.localScreen" />
+			<CallButton v-if="!isInCall"
+				shrink-on-mobile
+				:hide-text="isSidebar" />
```

Comment:
there is no TopBar in sidebar, when isInCall is false

```suggestion
			<CallButton v-if="!isInCall" shrink-on-mobile />
```